package/mu_nsist.py

```diff
@@ -34,20 +33,6 @@
 logger = logging.getLogger(__name__)
 
 
-def find_makensis_win():
-    """Locate makensis.exe on Windows by querying the registry"""
-    try:
-        nsis_install_dir = winreg.QueryValue(
-            winreg.HKEY_LOCAL_MACHINE, "SOFTWARE\\NSIS"
-        )
-    except OSError:
-        nsis_install_dir = winreg.QueryValue(
-            winreg.HKEY_LOCAL_MACHINE, "SOFTWARE\\Wow6432Node\\NSIS"
-        )
-
-    return pjoin(nsis_install_dir, "makensis.exe")
-
-
```

Comment:
Being ignorant of `pynsist`'s inner workings, I wonder why this is no longer needed. :)